Source/RSP/RSP Command.c

```diff
@@ -29,7 +29,7 @@
 #include <stdlib.h>
 
 #include "opcode.h"
-#include "RSP.h"
+#include "Rsp.h"
 #include "CPU.h"
 #include "RSP Registers.h"
```

Comment:
With a little re-installation of Windows, I'm sure it's conceivable, but as the project is in the hands of the open-source community I felt that any other contributors would be happy lending a hand.  Besides the fact that these changes get PJ64 portable, they're clean-ups as well for the better.